maven-plugins/run-automation-package-sample/README.md

```diff
@@ -0,0 +1,7 @@
+TODO: fill readme
```

Comment:
I would add something based on the description section of the knowledgebase

Reply:
and describe what need to configure (i.e. For EE, the token with some ref to the doc if required).